_posts/2021-01-18-changelog-139.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #139"
+date: 2021-01-18 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6497 -->
+* Take into account attributes under `#[cfg_attr]` attribute in our type inference, name resolution, and other analysis.
+  Now our code insight works much better with code containing attributes like `#[cfg_attr(..., derive(...))]`, `#[cfg_attr(..., path="...")]`, etc.
+  TODO: gif
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5913 -->
+* Introduce rename refactoring suggestions for local variables.
+  Now you will see a special hint to rename all usages when changing a local variable name in the editor (by [@Kobzol])
+  TODO: gif
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6582 -->
+* Add basic support for [overridden](https://rust-lang.github.io/rustup/overrides.html) toolchains.
+  Note, multiple overridden toolchains in a single project are still unsupported
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6553 -->
+* Resolve URLs in `Cargo.toml` (e.g. in dependency `git` links).
+  Now you can open URL specified in `git`, `homepage`, `repository`, and `documentation` fields (by [@Kobzol])
+  TODO: gif?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6084 -->
+* Provide folding for TOML files. Currently, it supports tables, arrays, and custom folding regions (by [@Kobzol])
+  TODO: gif?
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6120 -->
+* Highlight [string-formatting](https://doc.rust-lang.org/rust-by-example/hello/print/fmt.html) arguments of `panic!` macro
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6439 -->
+* Incrementally update our experimental [local crates index](https://github.com/intellij-rust/intellij-rust/pull/6390).
+  This speeds up the experimental `Cargo.toml` dependency completion. In the future, this would affect other new code insight based on the local crates index as well.
+  Note, the corresponding feature is disabled by default for now.
+  You can enable it via `org.rust.crates.local.index` option in `Experimental Feature` dialog (`Help | Find Action` and type `Experimental features` to open the dialog)
+
+## Fixes
+* Cargo features and `cfg` support improvements:
```

Comment:
This block looks more as a set of features than a set of fixes. I suppose we can move it into `features` section and add the first ite about `cfg_attr` into the same block

Reply:
I totally agree